app/views/lists/show.html.erb

```diff
@@ -44,16 +27,23 @@
 <% end %>
 
 <%= form_with(model: [ @list, @list.items.build ], local: true) do |form| %>
-  <p>
-    <%= form.text_field :name, placeholder: :name %>
-    <%= form.label :quantity %>
-    <%= form.number_field :quantity, value: 1 %>
-    <%= form.select(:category_id, options_for_select(@categories.map { |c| [c.name, c.id] })) %>
-    <%= form.submit "Add item" %>
-  </p>
+  <div class="row">
+    <div class="columns small-9 large-4">
+      <%= form.text_field :name, placeholder: :name %>
+    </div>
+    <div class="columns small-3 large-2">
+      <%= form.number_field :quantity, value: 1 %>
+    </div>
+    <div class="columns small-6 large-3">
+      <%= form.select(:category_id, options_for_select(@categories.map { |c| [c.name, c.id] })) %>
+    </div>
+    <div class="columns small-6 large-3">
+      <%= form.submit "Add item", class: "button small" %>
```

Comment:
Better rails style to use single quotes here

Reply:
Done in 82a524a (+ other instances)